tbx/project_styleguide/templates/patterns/pages/work/historical_work_page.html

```diff
@@ -77,7 +77,7 @@ <h1 class="heading heading--one-b">{{ page.title }}</h1>
 
             <ul class="streamfield__related-posts">
```

Comment:
It would be easiest if you don't make changes to this file, as it has changed considerably for the front-end. Also the `tag_link_base` is now "../"

Reply:
My changes have now been merged so you can make any updates to this template directly on the new version.